README.md

```diff
@@ -1,274 +1,62 @@
 # Auth0 Deploy CLI
 
-Auth0 supports continuous integration and deployment (CI/CD) of Auth0 Tenants through our [source control extensions](https://auth0.com/docs/extensions#deploy-hosted-pages-rules-and-database-connections-scripts-from-external-repositories) and integration into existing CI/CD pipelines by using this **auth0-deploy-cli** tool.
+Auth0 supports continuous integration and deployment (CI/CD) of Auth0 Tenants and integration into existing CI/CD pipelines by using this **auth0-deploy-cli** tool.
 
 The `auth0-deploy-cli` tool supports the importing and exporting of Auth0 Tenant configuration data.
 
-Supported Features
-
-- Supported Auth0 Objects
-  - Tenant Settings
-  - Rules (Including Secrets/Settings)
-  - Hooks
-  - Connections
-  - Custom Databases
-  - Clients / Applications
-  - Resource Servers (APIs)
-  - Pages
-  - Email Templates and Provider
-  - Guardian Settings
-- Configuration options
-  - Defined Directory Structure
-  - YAML Configuration
-  - Programmatically
-- Environment Variable Replacements
+Supported Auth0 Management API resources
+- [x] [Actions](https://auth0.com/docs/api/management/v2/#!/Actions/get_actions)
+- [ ] [Branding](https://auth0.com/docs/api/management/v2/#!/Branding/get_branding)
```

Comment:
q: do you know why we had handlers for brandings and prompts in `auth0-source-control-extensions-tool` but we don't support it in `deploy-cli`?

Reply:
Not really. We should review it with the team.